CONTRIBUTING.md

```diff
@@ -6,10 +6,10 @@ Lütfen dikkat: Bu projeye katkı sağlarken [Contributor Code of Conduct](https
 
 Eklerken, repodaki düzeni korumak adına bir takım basit kuralın hem bizlere hem de yeni gelecek arkadaşlara yardımcı olacağını düşündüm. O yüzden katkı sağlamak istiyorsanız, lütfen:
 
-- Repository'i kendi hesabınız üzerindne forklayın
+- Repository'i kendi hesabınız üzerinden fork'layın.
```

Comment:
Forklamak burada türkçeleşmiş bir fiil gibi davrandığı ve özel isim olmadığı için, üstten tire ile ayrılmasına gerek kalmıyor.